packages/@o3r/localization/src/tools/localization.service.ts

```diff
@@ -49,7 +48,7 @@ export class LocalizationService {
   /**
    * Internal subject that we use to track changes between keys only and translation mode
    */
-  // eslint-disable-next-line @typescript-eslint/naming-convention
+  // eslint-disable-next-line @typescript-eslint/naming-convention -- private field
   private readonly _showKeys$ = new BehaviorSubject(false);
```

Comment:
```suggestion
  private readonly showKeys = new BehaviorSubject(false);
```

Reply:
One of our recommendation is to have a suffix `$` when using observable or subject no ?

Reply:
I don't think we recommend this for Subject, only for Observables.

Reply:
Indeed the private field name can be changed (or the rule adapted) to not disable `no-underscore-dangle` in several places

Reply:
Fixed with the new config